linux_os/guide/system/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_chmod/rule.yml

```diff
@@ -26,6 +26,9 @@ rationale: |-
 
 severity: medium
 
+platforms:
+    - not aarch64_arch
```

Comment:
I finally managed a `ppc64le` system to test these rules and noticed they were not returning `not applicable`. And it is because a different architecture is used here. The platform is filtering by `aarch64`:
`{ ! ( grep -q aarch64 /proc/sys/kernel/osrelease ); }`

It should filter by `ppc64le`:
```
$ cat /proc/sys/kernel/osrelease
4.18.0-513.el8.ppc64le
```